src/js/actions/libraries.js

```diff
@@ -354,13 +355,19 @@ define(function (require, exports) {
                     return descriptor.playObject(placeObj);
                 })
                 .then(function () {
-                    // Standard PS will auto select the newly placed layer. This gives us the opportunity 
-                    // to get the new layer ID by getting the "layerID" property of the currently selected layer. 
-                    return descriptor.getProperty("layer", "layerID");
+                    return this.flux.actions.layers._getLayerIDsForDocumentID(currentDocument.id);
```

Comment:
You definitely want to transfer here instead of enqueueing this action. In this case, you were lucky that we didn't get a dead lock from locks, but returning an enqueue is a bad idea.